### PR TITLE
add robot tools to care-o-bot-robot variant

### PR DESCRIPTION
**Pull request by floweisshardt**
requires https://github.com/ros/rosdistro/pull/13933

### CODE REVIEW

File: care_o_bot_robot/package.xml [hunk 1]
```diff
@@ -13,7 +13,13 @@
   <exec_depend>cob_bringup</exec_depend>
   <exec_depend>cob_manipulation</exec_depend>
   <exec_depend>cob_navigation</exec_depend>
+  <exec_depend>htop</exec_depend>
+  <exec_depend>openssh-server</exec_depend>
+  <exec_depend>python-wstool</exec_depend>
   <exec_depend>robot_upstart</exec_depend>
+  <exec_depend>tmux</exec_depend>
+  <exec_depend>tree</exec_depend>
+  <exec_depend>vim</exec_depend>
```
Comment (fmessmer) [inline care_o_bot_robot/package.xml:22]: I don't like this approach too much...
Can we add a new meta-package e.g. `care_o_bot_tools` within this repo which defines the `exec_depend` to those system tools? And then we have `care_o_bot_robots` depend on `car_o_bot_tools`...

**Review by fmessmer — DISMISSED** [review]

File: care_o_bot_robot/package.xml (hunk 1 above)
Comment (floweisshardt) [reply care_o_bot_robot/package.xml:22]: What would be the difference?

**Comment by floweisshardt** [comment]
make sure to add everything from setup_cob4 `InstallCob4.sh` script:

```
sudo apt-get install vim tree gitg git-gui meld curl openjdk-6-jdk zsh terminator language-pack-de language-pack-en ipython nmap -y --force-yes
sudo apt-get install --install-recommends linux-generic-lts-wily -y --force-yes
sudo apt-get install openssh-server -y --force-yes
sudo apt-get install ros-indigo-rosbash -y --force-yes
sudo apt-get install ntp -y --force-yes
sudo apt-get install apt-cacher-ng
sudo apt-get install nfs-kernel-server autofs -y --force-yes
sudo apt-get install zlib1g-dev uuid-dev libmnl-dev gcc make git autoconf autoconf-archive autogen automake pkg-config curl -y --force-yes
```

File: care_o_bot_robot/package.xml (hunk 1 above)
Comment (fmessmer) [reply care_o_bot_robot/package.xml:22]: Ok then, I approve to add them here...
still please consider https://github.com/ipa320/care-o-bot/pull/38#issuecomment-303103713

**Review by fmessmer — CHANGES_REQUESTED** [review]
Structure approved....please add missing tools from https://github.com/ipa320/care-o-bot/pull/38#issuecomment-303103713

**Comment by fmessmer** [comment]
I tested this with the fresh system for kinetic...
But `rosdep install --from-paths src -i -y` does not install packages like `htop`, 'blender', ....
It installs all other dependencies listed in `src`...but not those...don't know why and also do not no how to debug....

**Comment by fmessmer** [comment]
@ipa-fmw according to @ipa-mdl `exec_depend` entries are not considered by `rosdep install` for `meta-packages`...

**Comment by mathias-luedtke** [comment]
> according to @ipa-mdl exec_depend entries are not considered by rosdep install for meta-packages...

I meant that somehow meta-packages don't get processed at all..

**Comment by fmessmer** [comment]
fixed in #48

**Pull request closed without merging**